code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -666,21 +666,15 @@
 
 			if ("tip")
 				if (!src.restrained() && !src.stat)
+					if (istype(src.head, /obj/item/clothing/head/mj_hat || /obj/item/clothing/head/det_hat/))
+						src.say (pick("M'lady", "M'lord", "M'liege")) //male, female and non-binary variants with alliteration
+						//maptext_out = "<I>tips their fedora</I>"
```

Comment:
commented out code?

Reply:
Have an issue where the map text seemingly should appear because the spacing for the appears but no map text actually shows up. would be nice to return to it and fix it but it's not necessary